pctasks/run/pctasks/run/workflow/executor/remote.py

```diff
@@ -557,53 +577,22 @@ def complete_job_partitions(
                                         status=JobPartitionRunStatus.FAILED,
                                     )
+                                completed_job_count += 1
+                                logger.info(
+                                    f"Job partition completed: {job_id}:{part_id}"
+                                )
 
-                                # Handle job completion
-                                if (
-                                    job_part_state.current_task is None
-                                    and job_part_state.status
-                                    != JobPartitionStateStatus.FAILED
-                                ):
-                                    try:
-                                        job_part_state.status = (
-                                            JobPartitionStateStatus.SUCCEEDED
-                                        )
-                                        update_job_partition_run_status(
-                                            container,
-                                            run_id=run_id,
-                                            job_partition_run_id=part_run_record_id,
-                                            status=JobPartitionRunStatus.COMPLETED,
-                                        )
-                                        self.handle_job_part_notifications(
-                                            job_part_state
-                                        )
-                                    except Exception:
-                                        job_part_state.status = (
-                                            JobPartitionStateStatus.FAILED
-                                        )
-                                        failed_job_count += 1
-                                        update_job_partition_run_status(
-                                            container,
-                                            run_id=run_id,
-                                            job_partition_run_id=part_run_record_id,
-                                            status=JobPartitionRunStatus.FAILED,
-                                        )
-                                    completed_job_count += 1
-                                    logger.info(
-                                        f"Job partition completed: {job_id}:{part_id}"
-                                    )
-
-                                _report_status()
+                            _report_status()
 
-                    time.sleep(0.25 + ((random.randint(0, 10) / 100) - 0.05))
+                time.sleep(0.25 + ((random.randint(0, 10) / 100) - 0.05))
 
-                logger.info(f"Partition group {group_id} completed!")
+            logger.info(f"Partition group {group_id} completed!")
 
-            except Exception as e:
-                logger.exception(e)
-                raise
+        except Exception as e:
+            logger.exception(e)
```

Comment:
The first argument to `Logger.exception` is the message. It will automatically include the exception info.

```suggestion
            logger.exception()
```

Reply:
Not supplying the msg argument gives me Pylance errors:

![image](https://user-images.githubusercontent.com/2320142/220481910-49cac5ec-7437-47a4-87d0-ee100e92e26c.png)

Given this, I think calling with the "e" as the msg makes sense.